### PR TITLE
Added Synonym-Searcher

### PR DESCRIPTION
**Pull request by AnshikaAg03**
Closes: #452 

- **Title** : Synonym-Searcher
- **Name:** : Anshika
- **Idenitfy yourself:** : GSSOC

### Describe the add-ons or changes you've made 📃
 I have added a synonym-searcher.

### How Has This Been Tested? ⚙️

I have tested this in my local computer

### Screenshots 📷

image

### Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by AnshikaAg03** [comment]
@Avdhesh-Varshney  please check my pr now , i have made all the requested changes

File: Vanilla-JS-Projects/Intermediate/Synonym-Searcher/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- Fork this repository.
+- Clone the forked repository.
+- Open index.html in your web browser to start your culinary exploration.
+
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+![image](./Synonym-Searcher.webp)
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Synonym-Searcher/README.md:54]: change it and attach `screenshot.webp`

File: Vanilla-JS-Projects/README.md [hunk 2]
```diff
@@ -75,6 +75,7 @@
 |  48   |  [Language-Learning-Website](./Intermediate/Language-Learning-Website/) | ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
 |  49   | [Valid-Parenthesis-Checker](./Intermediate/Valid-Parenthesis-Checker/) | ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
 </div>
+|  50  | [Synonym-Searcher](./Intermediate/Synonym-Searcher/) | ![Intermediate](https://img.shields.io/badge/Intermediate-FFD700?style=for-the-badge) |
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/README.md:78]: keep it inside the div not outside.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by AnshikaAg03** [comment]
@Avdhesh-Varshney I have made the requested changes please review it now

File: Vanilla-JS-Projects/Intermediate/Synonym-Searcher/README.md (hunk 1 above)
Comment (Avdhesh-Varshney) [reply Vanilla-JS-Projects/Intermediate/Synonym-Searcher/README.md]: image

**Comment by AnshikaAg03** [comment]
@Avdhesh-Varshney  i have uploaded screenshot.webp now

**Review by Avdhesh-Varshney — APPROVED** [review]

**Pull request merged**